src/App.tsx

```diff
@@ -1,20 +1,31 @@
-import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
+import { Navigate, Route, Routes } from 'react-router-dom';
 
 import './App.scss';
+import { NavBar } from './components/NavBar';
+import { HomePage } from './components/HomePage';
+import { NotFoundPage } from './components/NotFoundPage';
+import { PeoplePage } from './components/PeoplePage';
 
 export const App = () => {
   return (
     <div data-cy="app">
-      <Navbar />
+      <NavBar />
 
-      <div className="section">
+      <main className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="home" element={<Navigate to="/" replace />} />
+            <Route path="*" element={<NotFoundPage />} />
+
+            <Route path="people">
+              <Route index element={<PeoplePage />} />
+              <Route path=":slug" element={<PeoplePage />} />
+            </Route>
+
+          </Routes>
```

Comment:
```suggestion
            </Route>
          </Routes>
```
Don't leave empty lines between parent and child components